browser/ui/toolbar/brave_app_menu_model.cc

```diff
@@ -247,12 +247,6 @@ void BraveAppMenuModel::InsertBraveMenuItems() {
   }
 #endif
 
-  // Insert adblock menu at last. Assumed this is always enabled.
-  DCHECK(IsCommandIdEnabled(IDC_SHOW_BRAVE_ADBLOCK));
```

Comment:
I think we don't need `IDC_SHOW_BRAVE_ADBLOCK` command id anymore?

Reply:
i see `IDC_SHOW_BRAVE_ADBLOCK` is used in couple of places still:

1. `brave_browser_command_controller_browsertest.cc`
2. `brave_browser_command_controller.cc`

Reply:
If no one use `IDC_SHOW_BRAVE_ADBLOCK` command, all the code in command_controller.cc will not be used.
As menu item was the only caller, I think we can safely delete all them.

Reply:
removed